process/smartContract/process.go

```diff
@@ -1160,10 +1172,18 @@ func (sc *scProcessor) DeploySmartContract(tx data.TransactionHandler, acntSnd s
 		return 0, err
 	}
 
-	err = sc.updateDeveloperRewards(tx, vmOutput, 0)
-	if err != nil {
-		log.Debug("updateDeveloper rewards error", "error", err.Error())
-		return 0, err
+	if !sc.flagDeploy.IsSet() {
```

Comment:
maybe have a function of updateDeveloperRewardsProxy -> which does this if / else inside

Reply:
done